examples/getting-started-movielens/03-Training-with-PyTorch.ipynb

```diff
@@ -37,45 +37,36 @@
     "\n",
```

Comment:
<p>Do we get information from the schema file for the output dimensions?</p><p> </p><p>Otherwise, we can add custom code for it. <code>nvt.ops.get_embedding_sizes</code> is not really used in any other library (I guess).</p><p> </p><p>If we use the data from schema and/or custom code, we might can simplify the 2-3 cells above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>yup, that is a great point! grabbing the information from the schema which simplified the code a lot</p>